frontend/simple/components/ProfilePicture.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+ <img :src="picture">
+</template>
+<script>
+import {HapiNamespace} from '../js/backend/hapi'
+import {latestContractState} from '../js/state'
+var namespace = new HapiNamespace()
+export default{
```

Comment:
Needs a space. This would have failed the linter, I think. Just opened related issue #253 to make linter check .vue files.